File: cpp/2nd_Week/Multiple Inheritance/Bangunan.cpp
#include <iostream>
#include <string>
using namespace std;

class Bangunan
{
    private:
        int kodeBangunan;
        string pemilikBangunan;
    public:
        Bangunan(){
        };

        void setKodeBangunan(int kodeBangunan){
            this->kodeBangunan = kodeBangunan;
        }

        void setPemilikiBangunan(string pemilikBangunan){
            this->pemilikBangunan = pemilikBangunan;
        }

        int getKodeBangunan(){
            return this->kodeBangunan;
        }

        string getPemilikiBangunan(){
            return this->pemilikBangunan;
        }
        
        ~Bangunan(){
        };
};

File: php/7th_week/interface_aset/Bangunan.php
<?php

class Bangunan{

    private $kodeBangunan;
    private $pemilikBangunan;

    function __construct()
    {
        
    }

    function setKodeBangunan($kodeBangunan){
        $this->kodeBangunan = $kodeBangunan;
    }

    function getKodeBangunan(){
        return $this->kodeBangunan;
    }

    function setPemilikBangunan($pemilikBangunan){
        $this->pemilikBangunan = $pemilikBangunan;
    }

    function getPemilikBangunan(){
        $this->pemilikBangunan;
    }

    function __destruct()
    {
        
    }
}

?>